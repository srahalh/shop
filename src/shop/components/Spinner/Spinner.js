import React from 'react';
import "./Spinner.scss";

export const Spinner = () => <div className="Spinner"></div>;

export default Spinner;
