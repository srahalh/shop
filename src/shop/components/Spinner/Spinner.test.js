import { render } from "@testing-library/react";
import React from "react";
import Spinner from "./Spinner";

describe("Spinner component", () => {
  test("Should render Spinner Component", () => {
    const { asFragment  } = render(<Spinner/>);
    expect(asFragment ()).toMatchSnapshot();
  });
});

