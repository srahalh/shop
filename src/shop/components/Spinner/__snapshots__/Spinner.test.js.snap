// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component Should render Spinner Component 1`] = `
<DocumentFragment>
  <div
    class="Spinner"
  />
</DocumentFragment>
`;
